eng/scripts/DevOps-Recordings-Start.ps1

```diff
@@ -0,0 +1,25 @@
+param ([Parameter(Mandatory=$True)]$pr, [Parameter(Mandatory=$True, ValueFromRemainingArguments=$true)][string[]]$sdks)
```

Comment:
You should put types on these as well, and attribute the ones above like you did here.